internal/version/version.go

```diff
@@ -5,36 +5,115 @@ package version
 
 import (
 	"encoding/json"
-)
+	"fmt"
+	"io"
 
-const minDriverVersion = "v0.0.21"
+	"gopkg.in/yaml.v3"
+)
 
+// the following variables are meant to be set at build time from 'ldflags'
 var (
-	BuildDate    string
-	BuildVersion string
-	GoVersion    string
+	// Major version
+	Major = ""
+	// Minor version
+	Minor = ""
```

Comment:
I'm curious why major and minor but not patch?

Reply:
Good call, added in 3d1660a2eac2078ae9c4fe60fb9bcff83bacd775